src/schema-association-service-1espt.ts

```diff
@@ -93,9 +88,40 @@ export async function getCached1ESPTSchema(context: vscode.ExtensionContext, org
     return undefined;
 }
 
-function disable1ESPTSchemaConfiguration() : void{
-    const config = vscode.workspace.getConfiguration('azure-pipelines');
-    config.update('1ESPipelineTemplatesSchemaFile', false, vscode.ConfigurationTarget.Workspace); // disable the 1ESPT schema configuration
-    vscode.window.showInformationMessage(Messages.disabled1ESPTSchemaAsADOOrgNotContains1ESPT);
+/**
+ * User is eligible for 1ESPT schema if 1ESPT schema is available in ADO organization
+ * @param azureDevOpsClient 
+ * @param organizationName 
+ * @returns 
+ */
+export async function checkIfUserEligibleFor1ESPTIntellisense(azureDevOpsClient: azdev.WebApi, organizationName: string): Promise<[boolean, string]> {
```

Comment:
You can simplify the return signature by just returning the repository ID (it'll be a string if it exists, undefined otherwise).
And then maybe the function can be renamed `get1ESPTRepoId` or something.